keyboards/system76/launch_2/launch_2.c

```diff
@@ -89,7 +89,10 @@ void bootmagic_lite(void) {
     // reset the EEPROM valid state and jump to bootloader.
     if ( matrix_get_row(0) & (1<<0) ) {
         eeprom_reset();
+        usb_mux_init();
```

Comment:
The usb_mux_init function could be called once prior to the if statement and eeprom_reset. It won't mind the timing of eeprom_reset.